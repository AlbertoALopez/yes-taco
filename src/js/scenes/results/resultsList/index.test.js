import React from 'react';
import ResultsList from './index.js';
import renderer from 'react-test-renderer';

test('ResultsList renders correctly', () => {
  const imageLabels = [{ description: 'dog' }];
  const tree = renderer.create(<ResultsList imageLabels={imageLabels} />).toJSON();

  expect(tree).toMatchSnapshot();
});
