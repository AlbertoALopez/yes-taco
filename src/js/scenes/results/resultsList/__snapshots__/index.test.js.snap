// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsList renders correctly 1`] = `
<div
  className="results-list-container"
>
  <ul
    className="results-list"
  >
    <li
      className="list-item"
    >
      <p>
        NaN
        % chance this is a 
        dog
        .
      </p>
    </li>
  </ul>
</div>
`;
